pages/your_tasks/TEMPLATE_your_tasks.md

```diff
@@ -1,56 +1,71 @@
 
-## Concrete problem 1, formulated as a question <!-- example: how to version control my code? -->
- 
+## Concrete problem 1, formulated as a question <!-- example: how to version control code? -->
+
 ### Description <!-- do not delete this heading and write your text below it -->
 
-Short explanation of what this problem is about.
+Short explanation of what the problem is about.
 
 ### Considerations <!-- do not delete this heading and write your text below it -->
 
-* [Structured in bullet points](style_guide#text) as much as possible, detailing things to consider about this problem in order to be able to find the right solution.
+Detail things to consider about this problem in order to be able to find the right solution - e.g. using a bullet point list.
+
+- Consideration 1
+- Consideration 2
+  - Sub-point
+- Consideration 3 
+  - Sub-point 
+  - Sub-point
 
 ### Solutions <!-- do not delete this heading and write your text below it -->
 
-By using [bullet point style](style_guide#text) as much as possible, try to describe when, why and for what is best to use a specific tool or resource. 
-Avoid making long list of links to tools and resources.
-Make sure to add the tools and resources mentioned in the text in the main "tools and resources" table.
+Briefly describe the use of specific tools or resources to solve the problem - e.g. using a bullet point list. 
+
+- Solution 1
+  - Sub-point
+- Solution 2
 
-* Bullet point solution 1
-  * Sub-point
-* Bullet point solution 2
+Make sure to add the tools and resources mentioned in the text in the "tools and resources" data file and then
+tag the page with such entries.
 
+Repeat the same structure for other problems you wish to mention on the page, as needed, e.g. as follows.
 
-## Concrete problem 2, formulated as a question <!-- example: how to name a software release? -->
+## Concrete problem 2, formulated as a question <!-- example: how to use GitHub to share code ? -->
  
 ### Description <!-- do not delete this heading and write your text below it -->
-Same as above
 
 ### Considerations <!-- do not delete this heading and write your text below it -->
-Same as above
 
 ### Solutions <!-- do not delete this heading and write your text below it -->
 
-## How to cite this page <!-- do not delete this heading and write your text below it -->
- contributors, page URL. Last date of access.
 
-## Tools and resources <!-- do not delete this heading and write your text below it -->
-List of relevant tools and resources for this task.
+## Concrete problem 3, formulated as a question <!-- example: how to use GitHub to share code ? -->
+
+### Description <!-- do not delete this heading and write your text below it -->
+
+### Considerations <!-- do not delete this heading and write your text below it -->
+
+### Solutions <!-- do not delete this heading and write your text below it -->
+
```

Comment:
```suggestion
```

The third example is a bit redundant, perhaps. I don't feel strongly about this, up to you.

Reply:
Yes, we did have 2 examples originally then I added another one - you are probably right, I'll shorten it, thx @egpbos.